src/command/branch.ts

```diff
@@ -53,7 +53,7 @@ export async function listBranch(repositoryPath: string, type: 'current' | 'loca
 
 export async function createBranch(repositoryPath: string, name: string, createOptions?: { startPoint?: string }, options?: IGitExecutionOptions): Promise<void> {
     const startPoint = createOptions ? createOptions.startPoint : undefined;
-    const args = ['branch', name];
+    const args = ['checkout', '-b', name];
```

Comment:
As this also checks out the branch, it is not really doing what the function name suggests.  I would suggest adding `checkout?: boolean` to createOptions, and do 'checkout -b' if set, 'branch' otherwise.  This also avoids any chance of breaking unknown users who rely of the branch not changing.

Reply:
I am OK with your proposal. Does that mean we would need to adjust it in Theia by adding the `checkout: true` option?

Reply:
If I'm not mistaken, [this](https://github.com/eclipse-theia/theia/blob/master/packages/git/src/browser/git-quick-open-service.ts#L290) is the only time the `createBranch` function is used in Theia. Adding here a `checkout` option by default set to `false` would not change the behavior in Theia. Setting the `checkout` to `true` and removing the call to `git.checkout` can be a nice optimization though.

Reply:
👍 Sounds good to me, please make the required changes. Also can you please bump up the version number (in the package.json) once this PR is ready, and we can make a new release.

Reply:
It should be ready now.